### PR TITLE
Update ranked score wins and losses

### PR DESCRIPTION
**Pull request by apple00juice**

### CODE REVIEW

**Review by paolovic — APPROVED** [review]

**Review by paolovic — COMMENTED** [review]
right now, I'm looking if there is another way of letting the registered users know...

File: src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java
Comment (paolovic) [inline src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java:250]: why not "running = newState"?
When we're in the if clause, newState must be false.

File: src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java
Comment (paolovic) [inline src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java:248]: setRunningToFalse or similar.
it's not a classic setter, we can just set it to false.

File: src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java
Comment (apple00juice) [reply src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java:250]: IDE, said to simplify

File: src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java
Comment (apple00juice) [reply src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java]: Changed it to setGameRunningToFalse, and did a check before the call. I think that makes more sense than that before.

**Comment by paolovic** [comment]
I think it's fine, can you merge #373 into here, before you merge this into master?

**Comment by apple00juice** [comment]
> > > I think it's fine, can you merge #373 into here, before you merge this into master?
> > 
> > 
> > sure, but why not directly merge 373 into master?
> 
> also good 👌🏻

ahh I just saw it is based on this one, right? Yea then we probably should merge it into this first. But feel free to do so

**Comment by paolovic** [comment]
> > > > I think it's fine, can you merge #373 into here, before you merge this into master?
> > > 
> > > 
> > > sure, but why not directly merge 373 into master?
> > 
> > 
> > also good 👌🏻
> 
> ahh I just saw it is based on this one, right? Yea then we probably should merge it into this first. But feel free to do so

Sorry, I was waiting for your review but forgot to assign you as a reviewer 😬.
That's why the delay...

File: src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java
Comment (hilalcomak) [inline src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java:248]: just an idea, how about endGame() instead of seGameRunningFalse()? (feel free to ignore this comment)

**Review by hilalcomak — APPROVED** [review]
thanks :)

File: src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java
Comment (apple00juice) [reply src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java:248]: yea thats even better

**Pull request merged**
